src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -30,5 +30,8 @@
     {
       "resourceType": "Email"
     }
-  ]
+  ],
+  "ConnectionStrings": {
+    "CatalogueUsers": "Data Source=.;Initial Catalog=CatalogueUsers;MultipleActiveResultSets=true;User ID=NHSD;Password=DisruptTheMarket1!"
```

Comment:
Will need to prompt Stuart & Adrian on this. I am guessing the ports will change within the development environment.